_posts/2022-12-01-athena.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title:  "Trino delivers for Amazon Athena"
+author: Manfred Moser, Martin Traverso
+image: /assets/trino-light.png
+excerpt_separator: <!--more-->
+---
+
+It is time to reach out to another large group of Trino users. The release of
+the new engine version for [Amazon Athena](https://aws.amazon.com/athena)
+upgrades Athena to a recent version of Trino from a rather old version. This
+update brings a ton of improvements from the Trino project to the users of the
+popular cloud-based query service.
+
+<!--more-->
+
+## Shared history
+
+Amazon Athena and Trino share a long history. From the beginning of Athena the
+query engine under the hood was Trino, then still called Presto. Athena created
+a low-maintenance, powerful access mode to your data in S3 and beyond. It
+combined the performance and features of Trino, with the convenience of a cloud
+service, and therefore enabled new users and use cases. You could take advantage
+of Trino, without needing a team of experts to deploy and operate a Trino
+cluster for your organization. In fact, we wrote about this in the first edition
+of **Trino: The Definitive Guide**. There is also a section in the [new second
+edition]({% post_url 2022-10-03-the-definitive-guide-2 %}), that you can get for
+[free from Starburst](https://www.starburst.io/info/oreilly-trino-guide/).
+
+## Time flies
+
+Since the initial release however, time has not stood still. In fact, the Trino
+project has accelerated in [innovation, features, and releases tremendously]({%
+post_url 2022-08-04-decade-innovation %}). Amazon Athena users now get access to
+many of these great features. As [AWS mentions in the
+announcement](https://aws.amazon.com/about-aws/whats-new/2022/10/amazon-athena-announces-upgraded-query-engine/),
+"over 50 new SQL functions, 30 new features, and more than 90 query performance
+improvements" are now available due the upgrade to a new version of Trino. These
+include [Row pattern recognition with MATCH_RECOGNIZE]({% post_url
+2021-05-19-row_pattern_matching %}), [new window features]({% post_url
+2021-03-10-introducing-new-window-features %}), support for ``UPDATE`` or
+``TRUNCATE`` statements, and many others.
+
+Performance improvements in our core engine as well as all the Trino connectors,
+show up in every release note. The [improvements observed by the Athena team in
+their
+benchmarks](https://aws.amazon.com/blogs/big-data/upgrade-to-athena-engine-version-3-to-increase-query-performance-and-access-more-analytics-features/)
+show the resulting gains nicely. This is great evidence that our approach of
+constantly working on small improvements, wherever we find potential, works
+well. This is necessary since we are already at a very high performance level,
+like an elite athlete, where every small improvement matters.
```

Comment:
```suggestion
well. This approach is necessary since Trino is at a very high performance level,
and like an elite athlete, every small improvement matters.
```